index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
There are two different functions for constructing Vine embed URLs and they depend on the location of the link. Twitter and certain Vine vids don't have an href in the element that we can use to construct the embed, so that's what the checkLocation thing is for. Let me know if you can think of another way to do that. I think just erasing the function will mean that we can't send to Min-Vid from certain contexts.